pkg/cluster/k8sres.go

```diff
@@ -250,6 +250,22 @@ func (c *Cluster) nodeAffinity() *v1.Affinity {
 	}
 }
 
+func (c *Cluster) tolerations() []v1.Toleration {
+	podToleration := c.Config.OpConfig.PodToleration
```

Comment:
I wonder, may be worth allowing "tolerations" field in postgresql manifest to overwrite default config, similar to other fields.

Adding tolerations to the top level of the manifest, would be aligned with e.g. resources also being there.

This would also allow specing more tolerations if needed, e.g. SSD or other custom settings specific to nodes.